upgrade-instructions.md

```diff
@@ -69,6 +69,28 @@ we regenerate invalid UidValidity upon reads.
 While this sanitizing is transparent to the end user and the admin, it might lead to rare IMAP client full mailbox
  resynchronisation. (one chance out of two billions).
 
+### Tune Cassandra time serie tables options
+
+Date 18/03/2020
+
+SHA-1 a6613bc91b c8792c64bf
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-3121
+
+Our usage of Cassandra for the time series have been improved by fine tuning the compaction strategy and
+the read repair option.
+
+#### Upgrade procedure
+
+Alter `deletedMailsV2` and `enqueuedMailsV3` with this `cqlsh` commands:
+
+```
+ALTER TABLE james_keyspace.deletedMailsV2 WITH compaction = { 'class' :  'TimeWindowCompactionStrategy'  };
+ALTER TABLE james_keyspace.enqueuedMailsV3 WITH compaction = { 'class' :  'TimeWindowCompactionStrategy'  };
+ALTER TABLE james_keyspace.deletedMailsV2 WITH read_repair_chance = 0.0;
+ALTER TABLE james_keyspace.enqueuedMailsV3 WITH read_repair_chance = 0.0;
+```
```

Comment:
Explain that compaction  strategies changes requires a compaction to actually take effect.

Reply:
@remk find a document explaining that it automatically triggers it

Reply:
What do you mean?

Why does it makes sense here?

Reply:
https://support.datastax.com/hc/en-us/articles/213370546-Change-CompactionStrategy-and-sub-properties-via-JMX

Doing via cqlsh with alter table trigger a compaction automatically on all nodes which can be on the heavy side, there is recommended alternatives using JMX but for this tables which should not grow too much the cqlsh approach seems good. There is no reference to having to trigger a compaction with nodetool in the documentation.
https://docs.datastax.com/en/cql-oss/3.3/cql/cql_reference/cqlAlterTable.html#cqlAlterTable__alter-compression